### PR TITLE
multi messages in the same TX

### PR DESCRIPTION
**Pull request by levackt**
Hey @assafmo , sorry I edited and merged the last one a bit hastily, opening this to try address your points in that commit. ​

The initial idea was to help the guys doing agent tips to do them faster than the tip bot does currently, not sure if it's useful afterall anyway.
But more generally, here we can either show many txs in  a block, many messages in a tx, or both as per this PR currently.

### CODE REVIEW

File: 4_transactions/send_batch.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ const main = async () => {
         if (currentHeight > startHeight) {
             break
         }
-        sleep(1)
+        console.log('Waiting for next block');
+        await sleep(1)
```
Comment (assafmo) [inline 4_transactions/send_batch.js:76]: `sleep(100)` would be more CPU friendly. :-)

**Review by assafmo — APPROVED** [review]
:fire:

**Comment by assafmo** [comment]
> sorry I edited and merged the last one a bit hastily

That's alright, I don't expect you to wait for me with PRs here. :slightly_smiling_face:

**Comment by assafmo** [comment]
That's awesome! I didn't know it's that easy to send multiple messages in the same tx using secretjs.
There's also a multiExecute function for Secret Contracts, it'd be great if you'll write an example for it.

For example:
```js
const gasPriceUscrt = 0.25; // fast
function getFeeForExecute(gas) {
  return {
    amount: [
      { amount: String(Math.floor(gas * gasPriceUscrt) + 1), denom: "uscrt" },
    ],
    gas: String(gas),
  };
}

// In the same tx: convert SCRT to sSCRT and send it to multiple people
secretjs.multiExecute(
  [
    // Convert 1 SCRT to 1 sSCRT
    {
      contractAddress: process.env.SSCRT_SNIP20_ADDRESS,
      handleMsg: { deposit: {} },
      transferAmount: [{ amount: String(1_000_000), denom: "uscrt" }],
    },
    // Then send 0.6 sSCRT to Alice
    {
      contractAddress: process.env.SSCRT_SNIP20_ADDRESS,
      handleMsg: {
        transfer: {
          recipient: process.env.ALICE_WALLET_ADDRESS,
          amount: String(600_000),
        },
      },
    },
    // Then send 0.4 sSCRT to Bob
    {
      contractAddress: process.env.SSCRT_SNIP20_ADDRESS,
      handleMsg: {
        transfer: {
          recipient: process.env.BOB_WALLET_ADDRESS,
          amount: String(400_000),
        },
      },
    },
  ],
  "My memo...",
  getFeeForExecute(130_000 + 150_000 * 2) // deposit usualy takes 130k gas and transfer takes 150k gas
);
```

File: 4_transactions/send_batch.js (hunk 1 above)
Comment (levackt) [reply 4_transactions/send_batch.js:76]: its converted to seconds but thanks

**Comment by levackt** [comment]
Good idea thanks, I didn't see multiExecute either. 
So will merge this one and make a new PR tomorrow with that, and bump the version to 0.11.0

**Pull request merged**